src/Optimization/hiopFactAcceptor.cpp

```diff
@@ -4,7 +4,7 @@
 //
 // This file is part of HiOp. For details, see https://github.com/LLNL/hiop. HiOp 
 // is released under the BSD 3-clause license (https://opensource.org/licenses/BSD-3-Clause). 
-// Please also read �Additional BSD Notice� below.
+// Please also read �Additional BSD Notice� below.
```

Comment:
there are some strange characters enclosing "Additional BSD Notice", please use enclosing quotes

Reply:
not fixed